app/services/batched_remove_status_service.rb

```diff
@@ -95,7 +85,7 @@ def unpush_from_public_timelines(status)
       redis.publish(status.local? ? 'timeline:public:local:media' : 'timeline:public:remote:media', payload)
     end
 
-    @tags[status.id].each do |hashtag|
+    status.tags.map { |tag| tag.name.mb_chars.downcase }.each do |hashtag|
```

Comment:
Creating two arrays when you could just do the name thing in the `each` block itself I guess.